packages/types/src/user.ts

```diff
@@ -54,6 +62,4 @@ type UpdateUserJSON = Pick<
   | 'unsafe_metadata'
 >;
 
-export type UpdateUserParams = Partial<
-  SnakeToCamel<UpdateUserJSON> & UpdateUserJSON
->;
+export type UpdateUserParams = Partial<SnakeToCamel<UpdateUserJSON>>;
```

Comment:
❓ Does this needs to be with `DeepSnakeToCamel`?
```suggestion
export type UpdateUserParams = Partial<DeepSnakeToCamel<UpdateUserJSON>>;
```